### PR TITLE
Hadoop FDW: per-session GUC was added for enable_limit_pushdown

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?
https://enterprisedb.atlassian.net/browse/FDW-690

### CODE REVIEW

File: product_docs/docs/hadoop_data_adapter/2/10d_example_limit_offset_pushdown.mdx [hunk 1]
```diff
@@ -83,3 +83,56 @@ __OUTPUT__
...
+QUERY PLAN
+----------------------------------------------------------------------------------
+ Limit
+   Output: empno, ename, job, mgr, hiredate, sal, comm, deptno
+   ->  Foreign Scan on public.emp
+         Output: empno, ename, job, mgr, hiredate, sal, comm, deptno
+         Remote SQL: SELECT * FROM `fdw_db`.`emp` ORDER BY `empno` ASC NULLS LAST
+(5 rows)
+
+SELECT * FROM emp ORDER BY empno LIMIT 5 OFFSET 2;
```
Comment (gvasquezvargas) [inline product_docs/docs/hadoop_data_adapter/2/10d_example_limit_offset_pushdown.mdx:103]: @sravanvelag  should this be a new input and output snippet starting here? Or is this still output from this query:  
```
SET hdfs_fdw.enable_limit_pushdown TO OFF;
EXPLAIN (VERBOSE, COSTS OFF)
SELECT * FROM emp ORDER BY empno LIMIT 5 OFFSET 2;
```

File: product_docs/docs/hadoop_data_adapter/2/10d_example_limit_offset_pushdown.mdx [hunk 2]
```diff
@@ -83,3 +83,56 @@ __OUTPUT__
...
+SELECT * FROM emp ORDER BY empno LIMIT 5;
+__OUTPUT__
+QUERY PLAN
+------------------------------------------------------------------------------------
+ Foreign Scan on public.emp
+   Output: empno, ename, job, mgr, hiredate, sal, comm, deptno
+   Remote SQL: SELECT * FROM `fdw_db`.`emp` ORDER BY `empno` ASC NULLS LAST LIMIT 5
+(3 rows)
+
+SELECT * FROM emp ORDER BY empno LIMIT 5;
```
Comment (gvasquezvargas) [inline product_docs/docs/hadoop_data_adapter/2/10d_example_limit_offset_pushdown.mdx:129]: @sravanvelag same here, I am not sure if this is a new query after enabling the GUC, or if it all still output from the query in lines 117-120.

Comment (sravanvelag) [reply product_docs/docs/hadoop_data_adapter/2/10d_example_limit_offset_pushdown.mdx:129]: same here too, this is a new query

**Review by nidhibhammar — APPROVED** [review]
LGTM

**Pull request merged**